File: java/link-message-client/src/main/java/link/message/client/content/complex/linear/LinearComplexMessageContentItem.java
package link.message.client.content.complex.linear;

import link.message.client.content.complex.ComplexMessageContentItem;

import com.alibaba.fastjson.annotation.JSONField;

/**
 * 负责消息里的线性消息项
 * 
 * @author zhongt
 *
 */
public class LinearComplexMessageContentItem extends ComplexMessageContentItem {
	@JSONField(name="display_type")
	protected int displayType;

	public int getDisplayType() {
		return displayType;
	}

	public void setDisplayType(int displayType) {
		this.displayType = displayType;
	}

}

File: net/Link.Message.Client/content/complex/linear/LinearComplexMessageContentItem.cs
using Newtonsoft.Json;

namespace Link.Message.Client.content.complex.linear
{
	/// <summary>
	/// 负责消息里的线性消息项
	/// </summary>
	public class LinearComplexMessageContentItem : ComplexMessageContentItem
	{
	    [JsonProperty("display_type")]
	    public int DisplayType { get; set; }
	}
}